content/3-lifecycle/1-on-mount/react/PageTitle.jsx

```diff
@@ -5,7 +5,7 @@ export default function PageTitle() {
 
 	useEffect(() => {
 		setPageTitle(document.title);
-	});
+	}, []);
```

Comment:
You don't need to specify an empty array of dependency for this use-case.
https://reactjs.org/docs/hooks-effect.html#recap